demos-europe/demosplan-core
# Fix planning document category resource type test

Description:
- Make work again PlanningDocumentCategoryResourceTypeTest

### How to review/test
- Run the test in local, all tests should pass
- The HttpRequestTest was removed and permission enabling moved to AbstractAPITest as same functionality is needed


Delete the checkbox if it doesn't apply/isn't necessary.

- [ x] Tests updated/created
